docs/data-cookbooks/run-tasks-in-lambda-or-docker.md

```diff
@@ -33,75 +33,222 @@ If your task requires more than any of these resources or an unsupported runtime
 
-```yaml
-    QueueGranules:
-      Type: Task
-      Resource: ${QueueGranulesLambdaFunction.Arn}
-```
+* A `aws_sfn_activity` resource:
 
-Given it has been discovered this task can no longer run in AWS Lambda, it can be run on the Cumulus ECS cluster by including the following in `app/config.yml`:
-
-```yaml
-  ecs:
-    instanceType: t2.large
-    desiredInstances: 1
-    services:
-      QueueGranules:
-        docker: true
-        image: cumuluss/cumulus-ecs-task:1.2.5
-        memory: 4000
-        count: 1
-        envs:
-          AWS_DEFAULT_REGION:
-            function: Fn::Sub
-            value: '${AWS::Region}'
-        commands:
-          - cumulus-ecs-task
-          - '--activityArn'
-          - function: Ref
-            value: QueueGranulesActivity
-          - '--lambdaArn'
-          - function: Ref
-            value: QueueGranulesLambdaFunction
-
-  activities:
-    - name: QueueGranules
+```hcl
+resource "aws_sfn_activity" "queue_granules" {
+  name = "${var.prefix}-QueueGranules"
+  tags = local.default_tags
+}
 ```
 
-and then modifying the corresponding Step Function State Machine definition in a workflow YAML file:
 
-```yaml
-    QueueGranules:
-      Type: Task
-      Resource: ${QueueGranulesActivity}
+* A instance of the `cumulus_ecs_service` module (found on the Cumulus [release](https://github.com/nasa/cumulus/releases page) configured to provide the QueueGranules task:
+
+```hcl
+
+module "queue_granules_service" {
+  source = "https://github.com/nasa/cumulus/releases/download/{version}/terraform-aws-cumulus-ecs-service.zip"
+
+  prefix = var.prefix
+  name   = "QueueGranules"
+  tags   = local.default_tags
+
+  cluster_arn                           = module.cumulus.ecs_cluster_arn
+  desired_count                         = 1
+  image                                 = "cumuluss/cumulus-ecs-task:1.3.0"
+  log2elasticsearch_lambda_function_arn = module.cumulus.log2elasticsearch_lambda_function_arn
+
+  cpu                = 400
+  memory_reservation = 700
+
+  environment = {
+    AWS_DEFAULT_REGION = data.aws_region.current.name
+  }
+  command = [
+    "cumulus-ecs-task",
+    "--activityArn",
+    aws_sfn_activity.queue_granules.id,
+    "--lambdaArn",
+    module.cumulus.queue_granules_task_lambda_function_arn
+  ]
+  alarms = {
+    TaskCountHight = {
+      comparison_operator = "GreaterThanThreshold"
+      evaluation_periods  = 1
+      metric_name         = "MemoryUtilization"
+      statistic           = "SampleCount"
+      threshold           = 1
+    }
+  }
+}
 ```
 
-## How do I name my resources? A short primer on Cumulus resource naming conventions
+* An updated [Discover Granules workflow](https://github.com/nasa/cumulus/blob/master/tf-modules/ingest/queue-granules-task.tf) to utilize the new resource (the resource key in the `QueueGranules` step has been updated to:
```

Comment:
Is this the URL you're intending to link here? Seems like you mean to link to the definition of the discover granules workflow in our example?

Reply:
Not at all. 

79640e732

Reply:
No, it's not. 

Fixed: f7142c038